pkg/apis/admission/types.go

```diff
@@ -63,6 +61,22 @@ type AdmissionReviewSpec struct {
 	UserInfo authentication.UserInfo
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// AdmissionResponse describes an admission response.
+type AdmissionResponse struct {
+	metav1.TypeMeta
+
```

Comment:
I think we do want to send back some of the metadata to ensure that the webhook is responding to the correct response (e.g., maybe for an audit log).

Reply:
It's too bad Object and OldObject are both inside the existing Spec object.